src/components/InputField/component.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getIn } from 'formik';
+import { Form, Input, Icon } from 'antd';
+
+const InputField = ({
+  placeholder,
+  autoComplete,
+  type,
+  icon,
+  field,
+  form: { touched, errors },
+  ...props
+}) => {
+  const fieldTouched = getIn(touched, field.name);
+  const fieldErrors = getIn(errors, field.name);
+
+  return (
+    <Form.Item
+      validateStatus={fieldErrors && fieldTouched && 'error'}
+      help={fieldErrors && fieldTouched && fieldErrors}
```

Comment:
You have duplicated `fieldErrors` here

Reply:
The second fieldErrors gives an error message to the help field.

Reply:
Fixed.